dstk-infra/apollo/src/graphql/model/model.ts

```diff
@@ -1,36 +1,42 @@
 import { builder } from '../../builder.js';
 import { Model } from 'objection';
 import { StorageProvider, ObjectionStorageProvider } from '../storage-provider/storageProvider.js';
-import { ObjectionMLModelVersion } from '../model-version/modelVersion.js';
+import { MLModelVersion, ObjectionMLModelVersion } from '../model-version/modelVersion.js';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const MLModel = builder.objectRef<any>('MLModel').implement({
+export const MLModel = builder.objectRef<ObjectionMLModel>('MLModel');
+
+builder.objectType(MLModel, {
     fields: (t) => ({
-        modelId: t.exposeID('modelId'),
+        modelId: t.field({
+            type: 'ID',
+            resolve(root: ObjectionMLModel, _args, _ctx) {
+                return root.$modelClass.idColumn[0];
+            },
+        }),
 
         storageProvider: t.field({
             type: StorageProvider,
-            async resolve(root, _args, _ctx) {
+            async resolve(root: ObjectionMLModel, _args, _ctx) {
                 const storageProvider = (await ObjectionStorageProvider.query()
-                    .findById(root.providerId)
+                    .findById(root.storageProviderId)
                     .first()) as typeof StorageProvider.$inferType;
                 return storageProvider;
             },
         }),
 
-        isArchived: t.exposeBoolean('isArchived'),
```

Comment:
I'd prefer we not delete this. Pothos makes an explicit choice to have this abstraction layer over top of the underlying data model, and I'd generally like to stay idiomatic wrt package conventions

Reply:
Oh sorry, that must have been a merge conflict mishap. Fixed.